### ISSUE TITLE
Support for Pocket API

### ISSUE THREAD

**Issue by jlelse** [CloudRail/cloudrail-si-android-sdk#16]
Please add support for read-it-later services like Pocket (https://getpocket.com/developer/)

**Comment by FlorianWendel** [comment]
We'll have a look at Pocket. Do you have any similar services from that category in mind?

**Comment by jlelse** [comment]
Yes Readability and Instapaper, but I don't know much about their APIs

**Comment by FlorianWendel** [comment]
Alright, we'll figure out if we can add support for those as well.
Thanks for the suggestion!

**Comment by butelo** [comment]
I have another suggestion: flickr :)
